uqcsbot/scripts/crates.py

```diff
@@ -315,16 +345,21 @@ def create_slack_divider_block() -> Dict[str, str]:
 
 
 def get_crate_blocks(crate: CrateResult) -> List[dict]:
-    """Converts a crate into its block based message format for posting to slack"""
+    """
+    Converts a crate into its block based message format for posting to slack
+    """
     return [
-        create_slack_section_block(TextBlock(f'*<{crate.homepage}|{crate.name}>*\n{crate.description}')),
+        create_slack_section_block(TextBlock(f'*<{crate.homepage}|{crate.name}>*'
+                                             f'\n{crate.description}')),
```

Comment:
\n should with the string of the line that's being ended

Reply:
Fixed.